### ISSUE TITLE
browser in windows build

### ISSUE THREAD

**Issue by esce76** [GrayHatter/uTox#182]
hi ! 

This is not stricte a issue. 
Can You add a option to manually select binary of web browser to view clicked links? 
(Windows build of uTox)

**Comment by tsudoko** [comment]
It should use the default browser set in the OS, does it not?

**Comment by esce76** [comment]
yes, works like U say, but i have 2 LUKSed partitions, and main daily browser is portable version of Iron and thats why i ask to....

**Comment by nCore** [comment]
+1 - there should be an option for specifying if you want to use system's default web browser or specified in kinda input text box

**Comment by gjedeer** [comment]
+1, I'm using a wrapper to run a browser via firejail and would like uTox to use that

**Comment by redmanmale** [comment]
I wrote the part about handling uri by custom application, but I've got couple questions about UI part.
UI would be like this:
![pic](http://puu.sh/y5frx/a876eeaccf.png)

But as I understand this option would be available only for Windows, so there should be a way to hide it on the other systems. In one of the previous PR's you said #if's are bad. So what is the proper way to do it?

@GrayHatter @robinlinden

**Comment by GrayHatter** [comment]
Don't hide it on the other systems. There's no reason to hide this feature from the other platforms, just make sure you're abstracting the native logic from uTox core, so that anyone on the other systems can implement their own launcher function.

That said, I don't really like this idea too much. It's a nice feature to have for a VERY small subset of users, but more complicated than I want uTox to be. It's just asking for additional errors/issues. My honest suggestion, do not implement this feature at all. However, if you think this is a feature we must have, use the --set command line flag. and store the information in the .ini file once that pull gets merged. You're gonna have to fight a hard/uphill battle to convince me this belongs in the UI.

**Comment by redmanmale** [comment]
uTox is all about security so if some users need that option I see no harm in letting them have it.
If you don't want to add this option to UI then at least we could have it in config (one flag on/off and one string with path to application).
